src/ProjectSystem/Impl/Project/FileSystemMirroringProjectSourceItemProviderExtensionBase.cs

```diff
@@ -118,7 +118,9 @@ public abstract class FileSystemMirroringProjectSourceItemProviderExtensionBase
         private async Task<ProjectItem> GetMsBuildItemByProjectItem(IProjectItem projectItem) {
             using (var access = await _projectLockService.ReadLockAsync()) {
                 var project = await access.GetProjectAsync(_configuredProject);
-                return project.GetItemsByEvaluatedInclude(projectItem.EvaluatedInclude).FirstOrDefault(pi => StringComparer.OrdinalIgnoreCase.Equals(pi.ItemType, projectItem.ItemType));
+                var item =  project.GetItemsByEvaluatedInclude(projectItem.EvaluatedInclude)
+                    .FirstOrDefault(pi => StringComparer.OrdinalIgnoreCase.Equals(pi.ItemType, projectItem.ItemType));
```

Comment:
Is it just formatting?
`pi => pi.ItemType.EqualsIgnoreCase(projectItem.ItemType)